pandas/tests/indexes/test_numeric.py

```diff
@@ -18,6 +18,16 @@
 from pandas.tests.indexes.common import Base
 
 
+# For testing division by (or of) zero for Series with length 5, this
```

Comment:
maybe you misunderstand what I want here

```
# gives several scalar-zeros and length-5 vector-zeros
 zeros = [box([0] * 5, dtype=dtype)
          for box in [pd.Index, np.array]
          for dtype in [np.int64, np.uint64, np.float64]]
 zeros.extend([np.array(0, dtype=dtype)
              for dtype in [np.int64, np.uint64, np.float64]])
 zeros.extend([0, 0.0, long(0)])

@pytest.mark.fixture(params=zeros)
def zero(request):
    return request.param
```

then remove the ``@pytest.mark.parmetrize`` on all the test functions

Reply:
See previous comment:

> Made zeros a fixture in test_numeric as requested, but that broke tests in test_range b/c of pytest magic

AFAICT the problem is caused by the fixture being in the namespace for test_numeric but not for test_range, which has a class that inherits from the test_numeric.Numeric (which defines the relevant tests).  I guess we could get around that by putting this in a shared conftest file, but all that accomplishes is making it harder for a reader to find out where things are defined.

Reply:
https://travis-ci.org/pandas-dev/pandas/jobs/335770424

Reply:
so move it to the conftest, this is quite standard practice and avoids code duplication.